src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -11,9 +11,10 @@
 using NuGet.Common;
 using NuGet.Commands;
 
-#if DEBUG
+// Uncomment the following when debugging (1 of 2). Also uncomment the PackageReference for Microsoft.Build.Locator.
+/*#if DEBUG
 using Microsoft.Build.Locator;
-#endif
+#endif*/
```

Comment:
You could consider deleting this entire block, and change the MSBuildLocator usage below to `Microsoft.Build.Locator.MSBuildLocator`, so it doesn't need a using statement, and therefore it's still a single block to uncomment for debugging.